tests/integration/helpers.py

```diff
@@ -232,6 +232,20 @@ def get_reachable_units(ops_test: OpsTest, app: str = APP_NAME) -> Dict[int, str
     return result
 
 
+async def unit_hostname(ops_test: OpsTest, unit_name: str) -> str:
```

Comment:
Why is this needed?

Reply:
We're using this hostname to cut network access via LXC in `helpers.cut_network_from_unit`